gspread/worksheet.py

```diff
@@ -1186,12 +1198,13 @@ def batch_update(
 
         body = {
             "valueInputOption": value_input_option,
-            "includeValuesInResponse": include_values_in_response,
             "responseValueRenderOption": response_value_render_option,
             "responseDateTimeRenderOption": response_date_time_render_option,
-            "data": data,
         }
 
+        body["includeValuesInResponse"] = include_values_in_response
+        body["data"] = data
```

Comment:
these two `body` modifications can go in the object now, instead of afterwards (they were not before because of parse_enum function)

Reply:
Yes it can too. I forgot with  all the back and forth

Reply:
done